pcdsdevices/beam_stats.py

```diff
@@ -75,3 +77,96 @@ def __init__(self, prefix, *, name, skip_small_moves=True, atol=None,
             self.atol = atol
         super().__init__(prefix, name=name, skip_small_moves=skip_small_moves,
                          **kwargs)
+
+
+class Lcls(BaseInterface, Device):
```

Comment:
Any opinions on whether this should be called `Lcls` or `LCLS`? I don't mind either way but I find my fingers messing up when I try to type `Lcls`

Reply:
Should I change it to LCLS? I like `LCLS`

Reply:
Changed it in 9ea2804, I think it should be `LCLS` because it is an Acronym :) - thank you

Reply:
I think it makes it easier to remember how to type it- I'm just used to seeing it in all caps. This might technically break a convention but I think it's better like that

Reply:
I think we're all in agreement here 👍